vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/model/AnnotationItemLabelMockPointOptions.java

```diff
@@ -0,0 +1,53 @@
+package com.vaadin.flow.component.charts.model;
+
+/**
+ * Contains coordinates for {@link AnnotationItemLabel}
+ */
+public class AnnotationItemLabelMockPointOptions
```

Comment:
I know the name of the class is related to the HC types, but I am worried about this one being a bit long.

Reply:
Renamed it to `AnnotationItemLabelPoint`.